src/engine/Value.v3

```diff
@@ -135,6 +135,7 @@ component Values {
 	def v_d(v: Value) -> u64 { return Value.F64.!(v).bits; }
 	def v_l(v: Value) -> i64 { return i64.view(Value.I64.!(v).val); }
 	def v_w(v: Value) -> u64 { return Value.I64.!(v).val; }
+	def v_v128(v: Value) -> (u64, u64) { return (Value.V128.!(v).low, Value.V128.!(v).high); }
```

Comment:
Can you factor out the cast here to a local variable?

Reply:
Yes sure.